app/views/step_by_step_pages/schedule.html.erb

```diff
@@ -0,0 +1,82 @@
+<%
+  links = [
+    {
+      text: 'Step by step publisher',
+      href: step_by_step_pages_path
+    },
+    {
+      text: @step_by_step_page.title,
+      href: step_by_step_page_path(@step_by_step_page)
+    },
+    {
+      text: 'Schedule'
+    }
+  ]
+%>
+
+<%= render 'shared/steps/step_breadcrumb', links: links %>
+
+<%= render 'shared/steps/heading',
+  step_nav: @step_by_step_page.title,
+  action: 'Set a date and time to publish'
+%>
+
+<%= render 'nav', step_by_step_page: @step_by_step_page, active: "schedule" %>
+
+<% if @schedule_intent %>
```

Comment:
Can you move the addition `@schedule_intent` to the commit where it's created? Generally, it's preferable all of the tests pass in each commit.

Reply:
Fair enough 👍 wasn't sure whether to put all the UI stuff in one commit, but your suggestion makes more sense.

Reply:
Actually I may just merge this commit and the next one, because you also want the `routes.rb` change to be in the same commit as the controller tests. That would just leave a half-finished `schedule.html.erb file` in this commit, which isn't very useful.

Reply:
I'd be wary of merging commits.

The new route should be added with the controller, but A rather than adding a full view, you can just add a placeholder one with `<p>To Do</p>`